### PR TITLE
improvement: apply all node style changes within same 'tick' at the same time

### PR DESCRIPTION
**Pull request by larsmoa**
Fixes an issue with timeline where the styling intermittently would be reset to the default styling

### CODE REVIEW

File: viewer/packages/cad-styling/src/NodeAppearanceProvider.test.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ describe('NodeAppearanceProvider', () => {
 
   beforeEach(() => {
     provider = new NodeAppearanceProvider();
-    jest.useFakeTimers();
+    jest.useFakeTimers('modern');
```
Comment (larsmoa) [inline viewer/packages/cad-styling/src/NodeAppearanceProvider.test.ts:17]:
```suggestion
    jest.useFakeTimers();
```

File: viewer/packages/rendering/src/rendering/NodeAppearanceTextureBuilder.test.ts [hunk 2]
```diff
@@ -17,11 +17,16 @@ describe('NodeAppearanceTextureBuilder', () => {
...
 
   beforeEach(() => {
+    jest.useFakeTimers();
     styleProvider = new NodeAppearanceProvider();
     builder = new NodeAppearanceTextureBuilder(1, styleProvider);
     nodeCollection = new TreeIndexNodeCollection([0]);
   });
 
+  afterEach(() => {
+    jest.useRealTimers;
```
Comment (haakonflatval-cognite) [inline viewer/packages/rendering/src/rendering/NodeAppearanceTextureBuilder.test.ts:27]: Should this be a function call?

File: viewer/packages/cad-styling/src/NodeAppearanceProvider.ts [hunk 3]
```diff
@@ -116,6 +117,10 @@ export class NodeAppearanceProvider {
   private notifyChanged() {
     this._events.changed.fire();
   }
+  /**
```
Comment (haakonflatval-cognite) [inline viewer/packages/cad-styling/src/NodeAppearanceProvider.ts:120]:
```suggestion

  /**
```

**Review by haakonflatval-cognite — APPROVED** [review]
Looks mostly good, a couple of comments

File: viewer/packages/rendering/src/rendering/NodeAppearanceTextureBuilder.test.ts (hunk 2 above)
Comment (larsmoa) [reply viewer/packages/rendering/src/rendering/NodeAppearanceTextureBuilder.test.ts:27]:
```suggestion
    jest.useRealTimers();
```

**Pull request merged**